include/zephyr/dt-bindings/clock/imx_ccm_rev2.h

```diff
@@ -99,4 +99,8 @@
 #define IMX_CCM_PIT_CLK                 0x5000UL
 #define IMX_CCM_PIT1_CLK                0x5001UL
 
+/* ADC */
+#define IMX_CCM_LPADC1_CLK	0x6000UL
+#define IMX_CCM_LPADC2_CLK	0x6001UL
```

Comment:
Hi @danieldegrasse, I know you enabled the FLEXSPI clock, I kept the indentation consistent with yours (FLEXSPI), but it seems inconsistent with the PIT.

Reply:
Sure- if you'd like to align the indentation that would be great. If possible, put the changes to fix indentation in a separate commit just to keep the history clean